dataactvalidator/config/fabsFields.csv

```diff
@@ -10,7 +10,7 @@ AwardingSubTierAgencyCode,awardingsubtieragencycode,awarding_sub_tier_agency_c,F
 AwardModificationAmendmentNumber,awardmodificationamendmentnumber,award_modification_amendme,FALSE,str,25,
 BusinessFundsIndicator,businessfundsindicator,business_funds_indicator,FALSE,str,3,
 BusinessTypes,businesstypes,business_types,FALSE,str,3,
-CFDA_Number,cfda_number,cfda_number,TRUE,str,7,
+AssistanceListingNumber,assistance_listing_number,assistance_listing_number,TRUE,str,7,
```

Comment:
There are no spaces in this word so the second column should be `assistancelistingnumber`